### DISCUSSION TITLE
Dubbi ottenimento OTP

### DISCUSSION THREAD

**Discussion by Sem98** [italia/ansc#743] [Q&A] [answered]
### Ambiente

pre-produzione

### Servizio

R007 - Servizi cooperativi firma ufficiale di stato civile

### idOperazione

Nessuno

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

9493

### Data del test (yyyy-mm-dd)

2024/3/1

### Descrivi il problema

Salve stavamo incominciando ad implementare lo sviluppo per la richiesta del json con l'otp reale dato che ancora usiamo quello di test "123456", non trovando documentazione al riguardo.
Leggendo tra le issue abbiamo visto che l'endpoint da richiamare è: "https://anscwebpre.anpr.interno.it/api-otp/usc/otp" , volevamo capire che payload bisogna passare al servizio per poter farci rispondere 200 con l'otp reale. 
Cordiali Saluti.

**Comment by mttfranci** [comment, marked as answer]
Buongiorno.

Gli OTP reali si possono ottenere solo previo login con SPID/CIE/CNS di un Ufficiale di Stato Civile dall'url : 

<https://anscweb.anpr.interno.it/ansc/login/otp>

Per maggiori dettagli potete fare riferimetn ao documento [Specifiche Tecniche Servizi Cooperativi JWT/JWS](https://github.com/italia/ansc/blob/main/docs/Note/SpecificheTecnicheServiziCooperativiJWT/index.md).

In particolare la versione del documento al momento in cui si scrive è la  [1.1.1](https://github.com/italia/ansc/blob/main/docs/Note/SpecificheTecnicheServiziCooperativiJWT/SpecificheTecnicheServiziCooperativiJWT_v1.1.1.pdf).
A pagina 5/6 trovate le indicazioni per la generazione dell'OTP in ambiente di produzione.

Saluti

Assistenza ANSC

PS: la issue è stata convertita in discussion.

**Comment by mttfranci** [comment]
@Sem98 si consiglia, per il futuro, di selezionare una delle ultime 3 voci dei template di creazione delle issue.
In questo caso specifico la voce "Dove trovo la documentazione?" sarebbe forse stata la più indicata.
Ciò è utile sia al fine di indirizzare più velocemente le richieste, ma permette anche a voi di non inserire una serie di informazioni non necessarie in questo caso.

![image](https://github.com/italia/ansc/assets/29305782/2c45108c-756d-4412-871c-57b28b0d778d)

Saluti

Assistenza ANSC
